### DISCUSSION TITLE
Logos for z/OS Open Tools?

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#60] [General]
Here's a few I came up with using an open source logo generator:
![logo](https://user-images.githubusercontent.com/39890068/197771441-f7c20ed7-4d3c-4ce8-ae23-ffac19475fb3.png)
![logo (1)](https://user-images.githubusercontent.com/39890068/197771450-b67ff288-9501-48e4-a9c2-bc44ce6263b9.png)
![logo (2)](https://user-images.githubusercontent.com/39890068/197771458-1bf04d94-a79e-47b6-a961-401916655d95.png)

If anyone is aware of other logo generators or can create a nice looking logo for z/OS Open Tools, then please chime in here!

**Comment by v1gnesh** [comment]
Hi,

How about a variation of the zOS Connect logo... except with a T01 model (double door), and the doors opened?
Doors open to signify open source, heh.

**Reply by MikeFultonDev** [reply]
sketch it and post it!

**Reply by v1gnesh** [reply]
Since I'm not a designer, I don't know about and don't have access to design-y tools.
This would take minutes for someone in IBM design (maybe the team that did the zOS Connect logo) to cartoon-ize an open double door mainframe. Especially since those folks might also use stablediffusion/midjourney and such, for getting ideas sketched out rapidly these days.

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM maybe it's just me but i don't see any logos

**Reply by IgorTodorovskiIBM** [reply]
Hmm, I see them in both firefox and chrome

**Comment by lbdyck** [comment]
How about something along these lines:

![image](https://user-images.githubusercontent.com/42328411/214863623-979517bc-b2cb-42b0-bf98-ea725cc0c07c.png)

Where it stands for z Open Source Tools

My $0.01 (not worth $0.02) contribution to this discussion.

**Comment by mgrossmann** [comment]
Would prefer zOT over zOST….  😬

**Reply by lbdyck** [reply]
I like zOT - sounds like an anteater (which could be a good moniker)

**Comment by lbdyck** [comment]
I found this - apparently UCI (University of California Irvine) uses it: 
![image](https://user-images.githubusercontent.com/42328411/215117693-9e8bc5e8-00ff-4fe1-b341-4aa87c09a2a5.png)

**Comment by MikeFultonDev** [comment]
I like ZOT too 😁. I am not sure UCI would mind?

**Comment by lbdyck** [comment]
So use zOT instead of ZOT - the case is all important :)  And it's a different use.

I found ZOT is trademarked by 42 different submissions. However those I looked at were for specific uses and none came close to using as a logo for an open source project.

See https://tmsearch.uspto.gov/bin/showfield?f=toc&state=4810%3Aq1mpov.1.1&p_search=searchss&p_L=50&BackReference=&p_plural=yes&p_s_PARA1=&p_tagrepl%7E%3A=PARA1%24LD&expr=PARA1+AND+PARA2&p_s_PARA2=zot&p_tagrepl%7E%3A=PARA2%24COMB&p_op_ALL=AND&a_default=search&a_search=Submit+Query&a_search=Submit+Query if you're interested.

**Reply by lbdyck** [reply]
Many of the 42 are dead of cancelled. This is the only one that may come close https://tmsearch.uspto.gov/bin/showfield?f=doc&state=4810:q1mpov.3.14

**Comment by lbdyck** [comment]
The use of the lower case z fits with the IBM products z/VM, z/OS, z15, z16, etc. And I suspect IBM trademarked the lowercase z prefix (I could be wrong).

**Comment by IgorTodorovskiIBM** [comment]
![DALL·E 2023-01-28 20 29 00 - A logo of a slanted mainframe with data projecting out of it enclosed in a circle, in 3d isomorphic, transparent background](https://user-images.githubusercontent.com/39890068/215300084-c2efc7fa-6d14-4306-80dc-a9b0828ad755.png)
Created using DALL-E with prompt "A logo of a slanted mainframe with data projecting out of it enclosed in a circle, in 3d isomorphic, transparent background"

**Reply by MikeFultonDev** [reply]
Cool

**Reply by v1gnesh** [reply]
There was a site suggested by @karpathy on Twitter for generating StableDiffusion content in batches, and then picking ones we like to inform the generation of the next batch. It was some website that does this. That'll help to quickly generate options and pick ones that'll work out best.

**Comment by netskink** [comment]
Here is a play on z/OS and Open Source, where OS is squared.  No icon graphics to keep it simple.  IBM official blue color. Pity I don't have a font with the stripes.  That in itself might be a good logo.
<img width="185" alt="Screenshot 2023-02-06 at 3 37 56 PM" src="https://user-images.githubusercontent.com/4422172/217080294-dd0b2b3c-36b6-413d-92e6-4542523e63f4.png">

**Comment by MikeFultonDev** [comment]
I'm thinking 'Community' should be in the title if we go with text - such an important part. 
However, I'm also thinking an icon with no text at all might be nice - poking at the whole community thing - it makes it more universal (not English based?)
Thoughts?

**Comment by IgorTodorovskiIBM** [comment]
<img width="620" alt="zosopentools" src="https://github.com/ZOSOpenTools/meta/assets/39890068/74c7c97e-210d-432a-abf0-cfe0cfd33248">

**Comment by MikeFultonDev** [comment]
it would be nice to have an icon too

**Comment by minitree** [comment]
My two cents:
![zopentoolswh](https://github.com/ZOSOpenTools/meta/assets/24321826/f948a734-1648-47a1-b7ba-e9ee15ae6744)

Or the one with a more IBMish-vibe:
![zopentoolswhibm](https://github.com/ZOSOpenTools/meta/assets/24321826/22ad6272-8b3e-4f33-a1f3-6c6b6d638f45)

And the icon which gives out a little spooky vibe:
![zosopenicon256](https://github.com/ZOSOpenTools/meta/assets/24321826/9256e390-d0ff-470e-b694-5d550438b88e)

**Comment by IgorTodorovskiIBM** [comment]
Added a gradient to the above:
![zopen_grad1](https://github.com/ZOSOpenTools/meta/assets/39890068/322057d7-d5ab-4ec5-9bcf-f800fa933e4a)
![zopen_grad2](https://github.com/ZOSOpenTools/meta/assets/39890068/311a7c19-9d72-497d-ac53-1636b11399bb)

**Reply by lbdyck** [reply]
fix typos please :)

**Comment by lbdyck** [comment]
![image](https://github.com/ZOSOpenTools/meta/assets/42328411/e93b5b53-c983-4740-9409-460e1d1582f2)

**Reply by AnthonyGiorgio** [reply]
I like this one, but it should be a wire-frame z16, and the font kerning needs adjusting.

**Comment by minitree** [comment]
More of a box:
![g45](https://github.com/ZOSOpenTools/meta/assets/24321826/bc2c7b4e-b1b4-4911-aeb0-7a5d565f30f5)

Icon:
![zoticon](https://github.com/ZOSOpenTools/meta/assets/24321826/8ebab25f-0b46-4370-9227-b6e4931dd040)

**Reply by v1gnesh** [reply]
In my head, the open section of the box is a clever/silly way of hinting at open source.

**Reply by AnthonyGiorgio** [reply]
What if the text was on the top right corner of the logo?

**Reply by minitree** [reply]
Let's see: 
![cornertext](https://github.com/ZOSOpenTools/meta/assets/24321826/25cecaea-e18b-4690-9fa8-506dff716081)

**Reply by AnthonyGiorgio** [reply]
I like this better, but I'd probably iterate on the font site and placement a few times.

**Reply by v1gnesh** [reply]
@minitree can you share an svg of the first one.

**Reply by minitree** [reply]
@v1gnesh there it is -
![g45](https://github.com/ZOSOpenTools/meta/assets/24321826/4f796965-a262-42b9-a129-18ab890f1cc7)

**Comment by PJCatalano** [comment]
How about hand drawn submission. Make it look like a server rack, but open like a tool chest.

![IMG_3345](https://github.com/ZOSOpenTools/meta/assets/156842276/c6032acd-27f4-46db-b19c-8e4678a9d707)

**Reply by AnthonyGiorgio** [reply]
That's really cool.  I like it!

**Reply by lbdyck** [reply]
Just need a real picture of this and then the words can be added easily.

**Comment by IgorTodorovskiIBM** [comment]
Gemini can generates images as well now - https://gemini.google.com/

![Bard_Generated_Image](https://github.com/ZOSOpenTools/meta/assets/39890068/33224e8b-67b4-4c0a-b7bc-fdaff79496c7)

![Bard_Generated_Image2](https://github.com/ZOSOpenTools/meta/assets/39890068/e1507b22-653e-431e-8e9c-f9ec173021cf)

**Reply by AnthonyGiorgio** [reply]
That first image is looks like it's for a bar and grill on the pier.

**Comment by minitree** [comment]
More AI art:
![OIG4 r93z Ki](https://github.com/ZOSOpenTools/meta/assets/24321826/6bb77c44-6206-4d7c-ae79-a6bb78fa1a8a)

I like that it generates these little tool images.
